base/shared/src/main/scala/scalaz/typeclass/ShowInstances.scala

```diff
@@ -0,0 +1,15 @@
+package scalaz
+package typeclass
+
+trait ShowInstances {
+
+  implicit final def string: Show[String] =
+    Show.instance[String](identity)
+
+
+  implicit final def contravariant: Contravariant[Show] =
```

Comment:
I feel somewhat strange about having both `Show` instances for stdlib types and typeclass instances for `Show` in the same file, but I couldn't find precedent for how to do this. I personally think that `Show` is an excellent contravariant functor, so I put it in.

In general, I'm not sure how many stdlib instances should go here. I suspect `Int` and friends, but it raises the question of whether `1.toByte.show` should just be `"1"`, with no indication that it's a `Byte`. I personally like `Show` instances, where it's not contradicted by another goal, to make it as clear as possible what type is being shown.

Reply:
it's not just a contravariant, it's also a [`LabelledContravariantDerivez`](https://gitlab.com/fommil/scalaz-deriving/blob/v0.9.0/scalaz-deriving/src/main/scala/scalaz/Derivez.scala#L95-110) :wink: